hack/test-end-to-end.sh

```diff
@@ -307,7 +307,7 @@ docker push ${DOCKER_REGISTRY}/cache/ruby-20-centos7:latest
 echo "[INFO] Pushed ruby-20-centos7"
 
 echo "[INFO] Back to 'master' context with 'admin' user..."
-osc project default
+osc project "default/$(echo ${API_HOST} | sed 's/\./-/g'):${API_PORT}/system:admin"
```

Comment:
I don't want the url to nickname logic embedded in sed in a bash script... I'd rather grab the cluster admin's context on startup:
```
CLUSTER_ADMIN_CONTEXT=$(osc config view --flatten -o template -t '{{index . "current-context"}}')
```
and switch back to it
```
osc project $CLUSTER_ADMIN_CONTEXT
```